components/perl/IO-Socket-SSL/Makefile

```diff
@@ -0,0 +1,66 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2013 Alexander Pyhalov.  All rights reserved.
+# Copyright (c) 2022 Tim Mooney.  All rights reserved
+#
+
+BUILD_STYLE = makemaker
+BUILD_BITS = 32_and_64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =		IO-Socket-SSL
+COMPONENT_VERSION =		2.074
+IPS_COMPONENT_VERSION =		2.74
+COMPONENT_FMRI =		library/perl-5/io-socket-ssl
```

Comment:
Please make use of HUMAN_VERSION (here and in the manifest file). See archiver/lzop for an example use.

Reply:
Done.